docs/providers/azure/events/timer.md

```diff
@@ -33,7 +33,7 @@ functions:
   example:
     handler: handler.hello
     events:
-      - timer:
+      - timer: true
         name: timerObj #<string>, default - "myTimer", specifies which name is available on `context.bindings`
```

Comment:
This looks rather as nesting issue.

I believe right format would be:

```yaml
functions:
  example:
    handler: handler.hello
    events:
      - timer:
          name: timerObj #<string>, default - "myTimer", specifies which name is available on `context.bindings`
          schedule: 0 */5 * * * * #<string>, cron expression to run on
```

Reply:
hmm... just tested this indentation still seem to get the default settings....

Can you write in JSON what you're trying to do?
I believe its: `{timer: true, name: "somestr", schedule: "0 */5 * * * * "}`

What I have with timer true is what's working for me in production right now.

Reply:
In Framework convention is to construct events as:

```json
{ "timer": { "name": "somestr", "schedule": "..." } }
```

If you feel it's otherwise or have doubts, best if you move discussion to https://github.com/serverless/serverless-azure-functions as it's where Azure provider is implemented, and it's where all it's related concerns should be handled

Reply:
ahh alright I'll check that out and bring this over there when I have time thanks